### ISSUE TITLE
Play/Pause not working on Figgy audio player

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/figgy#4914]
Errors in JS console:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'getDuration')
    at CanvasInstance._hasRangeChanged (AVComponent.js:540:62)
    at CanvasInstance._lowPriorityUpdater (AVComponent.js:1253:22)
    at CanvasInstance.pause (AVComponent.js:1213:22)
    at HTMLButtonElement.<anonymous> (AVComponent.js:268:27)
```

Example: https://figgy.princeton.edu/catalog/41c0c084-a79b-4f68-ad46-197607afe1a6

## Sudden Priority Justification

We have students using these players for classes.

**Comment by hackartisan** [comment]
from @tpendragon: I had to temporarily disable figgy-web-prod-2 and figgy-web-prod-4 from the load balancer. There's a setting on the campus firewall that makes it so it won't block fast streaming audio requests which we had turned on in the past for figgy1/3 which must not be set, because the downloads get periodically cancelled for the other nodes if they're part of the cluster. It was making the audio player not play at all. I sent a ticket into lsupport.

This is a manual step we need to document for when we're provisioning new machines. Maybe have the figgy playbook print a message about this when it runs.
